yewprint-doc/Cargo.toml

```
@@ -10,31 +10,30 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.6", optional = true }
+gloo = "0.6"
+once_cell = "1.16"
 wasm-bindgen = "0.2"
 web-sys = { version = "0.3", features = ["Window", "MediaQueryList", "Event", "HtmlInputElement"] }
-gloo = "0.6"
-yew = "0.19"
-yew-router = "0.16"
-# yew = { git = "https://github.com/yewstack/yew", branch = "master" }
-# yew-router = { git = "https://github.com/yewstack/yew", branch = "master" }
-yewprint = { path = ".." }
-
 # `wee_alloc` is a tiny allocator for wasm that is only ~1K in code size
 # compared to the default allocator's ~10K. It is slower than the default
 # allocator, however.
 #
 # Unfortunately, `wee_alloc` requires nightly Rust when targeting wasm for now.
 wee_alloc = { version = "0.4.5", optional = true }
-
-# The `console_error_panic_hook` crate provides better debugging of panics by
-# logging them with `console.error`. This is great for development, but requires
-# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
-# code size when deploying.
-console_error_panic_hook = { version = "0.1.6", optional = true }
+# yew = { git = "https://github.com/yewstack/yew", branch = "master" }
+yew = "0.19"
+# yew-router = { git = "https://github.com/yewstack/yew", branch = "master" }
+yew-router = "0.16"
```

I don't know why there was commented dependencies... not useful

```suggestion
yew = "0.19"
yew-router = "0.16"
```